tests/drivers/flash/common/testcase.yaml

```diff
@@ -40,6 +40,19 @@ tests:
     integration_platforms:
       - qemu_x86
       - mimxrt1060_evk/mimxrt1062/qspi
+      - max32655evkit/max32655/m4
```

Comment:
this is not how integration_platforms should be used, the goal is to limit what get tested on PRs to a couple of targets only:

https://docs.zephyrproject.org/latest/develop/test/twister.html#:~:text=integration_platforms%3A%20%3CYML%20list%20of%20platforms/boards%3E